pkg/mud/game/game.go

```diff
@@ -15,15 +15,15 @@ type clientCommand struct {
 
 // Game defines the master game object and everything in the game
 type Game struct {
-	Chat        *chat.Chat
+	Chat        stream.Stream
```

Comment:
Should be a pointer as there is only one, and we want to be able modify the contents in the future